### ISSUE TITLE
CI bazel tests are failing with bazel 0.10.0 RC1

### ISSUE THREAD

**Issue by iirina** [bazelbuild/bazel#4429]
Reported error:
```
Failed actions:

[action:[configuration:[id:89e87be6cb5e4eff73d3ca9d2123100f], label://src/tools/android/java/com/google/devtools/build/android/desugar:desugar, stderr:[name:stderr, uri:file:///home/ci/.cache/bazel/_bazel_ci/eedbb4b85fb85f2c28394e60e308a4c4/execroot/io_bazel/bazel-out/_tmp/action_outs/stderr-362], type:Javac, commandLine:[external/local_jdk/bin/java, -Xbootclasspath/p:external/bazel_tools/third_party/java/jdk/langtools/javac-9-dev-r4023-3.jar, -jar, external/bazel_tools/tools/jdk/JavaBuilder_deploy.jar, @bazel-out/k8-fastbuild/bin/src/tools/android/java/com/google/devtools/build/android/desugar/libdesugar.jar-2.params]], id:[actionCompleted:[configuration:[id:89e87be6cb5e4eff73d3ca9d2123100f], label://src/tools/android/java/com/google/devtools/build/android/desugar:desugar, primaryOutput:/home/ci/.cache/bazel/_bazel_ci/eedbb4b85fb85f2c28394e60e308a4c4/execroot/io_bazel/bazel-out/k8-fastbuild/bin/src/tools/android/java/com/google/devtools/build/android/desugar/libdesugar.jar]]]



`bazel test` returned status 1

```

Full console log is [here](https://ci.bazel.build/job/Global/job/bazel-slow-tests/178/consoleFull).
This is blocking the 0.10.0 release (#3958).

**Comment by jin** [comment]
The fix is here: 65c13dd5a4c1b4b5a072f7680b8f1cf3c5079b52 @iirina cherrypick this, please?

**Comment by iirina** [comment]
Thanks @jin! I added it to the list of cherry picks.
